perl/dist.ini

```diff
@@ -0,0 +1,48 @@
+name      = Gherkin
+abstract  = A parser and compiler for the Gherkin language
+main_module = lib/Gherkin.pm
+author    = ['Peter Sergeant <pete@clueball.com>', 'Gaspar Nagy']
+license   = MIT
+is_trial  = 0
+perl      = 5.008
+copyright_holder = Peter Sergeant, Gaspar Nagy
+
+; Worth seeing the note about this in Gherkin.pm. You are aiming for the Gherkin
+; version with the minor revision %2d and a two-digit CPAN version added. For
+; example:
+;
+;   3.1.2 -> 3.1.0201, 3.1.0202, 3.1.0203 ... etc
+;   3.2.0 -> 3.2.0001, 3.2.0002 ... etc
+;
+; CPAN releases can't be changed, and this allows a nice simple way of uploading
```

Comment:
I note that the Gherkin release I've made is now on .0004. While this is due to incompetence on my part, I do wonder about this moving forward. CPAN is particularly unforgiving about allowing rereleases.

The problem I envisage is that you guys make a new Gherkin release, and for some Perl-related peculiarity, it breaks tests on certain machines. This then means it can't be installed on those machines. Do you really want to do a whole new central Gherkin release because the Perl version can't be installed on 50% of the machines due to some Perl weirdery?

In terms of the release process, running `make release` should transparently take care of all of the CHANGES related stuff, although as you correctly point out, that's an extra file to maintain.